ragaai_catalyst/tracers/agentic_tracing/tracers/llm_tracer.py

```
@@ -550,146 +550,139 @@ def create_llm_component(
+                "interactions": interactions,
+            }
+
+            # Assign context and gt if available
+            component["data"]["gt"] = span_gt
+            component["data"]["context"] = span_context
+
+            # Reset the SpanAttributes context variable
+            self.span_attributes_dict[name] = SpanAttributes(name)
+
+            return component
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)